lib/index.js

```diff
@@ -25,6 +25,8 @@ let selectedTaskIndex = null;
 let mode              = Mode.VIEW;
 let inputText         = '';
 
+
+
 try {
```

Comment:
Unnecessary empty lines. One is enough. Change is unrelated to the feature.
```suggestion
let inputText         = '';

try {
```

Reply:
resolved